src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/TypeWrapperTests.cs

```diff
@@ -44,256 +43,88 @@ public class ReferencedType
                 referencedImage = ms.ToArray();
             }
 
-            string source = @"
-            using System.Text.Json.Serialization;
-            using ReferencedAssembly;
+            string source = """
+                using System.Text.Json.Serialization;
+                using ReferencedAssembly;
 
-            namespace HelloWorld
-            {
-                [JsonSerializable(typeof(HelloWorld.MyType))]
-                [JsonSerializable(typeof(ReferencedAssembly.ReferencedType))]
-                internal partial class JsonContext : JsonSerializerContext
+                namespace HelloWorld
                 {
-                }
+                    [JsonSerializable(typeof(HelloWorld.MyType))]
+                    [JsonSerializable(typeof(ReferencedAssembly.ReferencedType))]
+                    internal partial class JsonContext : JsonSerializerContext
+                    {
+                    }
 
-                public class MyType
-                {
-                    public void MyMethod() { }
-                    public void MySecondMethod() { }
+                    public class MyType
+                    {
+                        public void MyMethod() { }
+                        public void MySecondMethod() { }
+                    }
                 }
-            }";
+                """;
 
             MetadataReference[] additionalReferences = { MetadataReference.CreateFromImage(referencedImage) };
 
             // Compilation using the referenced image should fail if out MetadataLoadContext does not handle.
             Compilation compilation = CompilationHelper.CreateCompilation(source, additionalReferences);
 
-            JsonSourceGenerator generator = new JsonSourceGenerator();
-
-            Compilation newCompilation = CompilationHelper.RunGenerators(compilation, out ImmutableArray<Diagnostic> generatorDiags, generator);
+            JsonSourceGeneratorResult result = CompilationHelper.RunJsonSourceGenerator(compilation);
 
             // Make sure compilation was successful.
-            Assert.Empty(generatorDiags.Where(diag => diag.Severity.Equals(DiagnosticSeverity.Error)));
-            Assert.Empty(newCompilation.GetDiagnostics().Where(diag => diag.Severity.Equals(DiagnosticSeverity.Error)));
+            Assert.Empty(result.Diagnostics.Where(diag => diag.Severity.Equals(DiagnosticSeverity.Error)));
+            Assert.Empty(result.NewCompilation.GetDiagnostics().Where(diag => diag.Severity.Equals(DiagnosticSeverity.Error)));
 
             // Should find both types since compilation above was successful.
-            Assert.Equal(2, generator.GetSerializableTypes().Count);
-        }
-
-        [Fact]
-        public void CanGetAttributes()
```

Comment:
Test deleted since the source generator no longer surfaces "TypeWrapper" values. The reflection wrappers will eventually get removed in a follow-up PR.